pkg/gui/presentation/commits.go

```diff
@@ -69,7 +69,7 @@ func GetCommitListDisplayStrings(
 	}
 
 	// this is where my non-TODO commits begin
```

Comment:
Is this comment still true then?

Also, "first non rebasing commit" is the (let's say) oldest commit, right?

Reply:
> Is this comment still true then?

Only conceptually, not technically. Fixed in ddfec40f2b5d.

> Also, "first non rebasing commit" is the (let's say) oldest commit, right?

No, the first one in the list (from the top), i.e. the newest one.